doc/whats_new/v0.23.rst

```diff
@@ -197,6 +197,11 @@ Changelog
 
 - |Enhancement| improve error message in :func:`utils.validation.column_or_1d`.
   :pr:`15926` by :user:`Loïc Estève <lesteve>`.
+
+- |Fix| `optimize._newton_cg` will now avoid an unnecessary iteration by checking for
```

Comment:
We also need to focus on public API in what's new. This is private. What public API did this affect?

Reply:
It affects LogisticRegression when arg `solver=newton-cg`. I am adding it in whatsnew

Reply:
done. Tell me if any change is required...